api/environment/env.go

```diff
@@ -59,6 +59,10 @@ type EnvDeleteResponse struct {
 	EnvResponse EnvDelete `yaml:"resp,omitempty" json:"resp,omitempty"`
 }
 
+type EnvUpdateResponse struct {
```

Comment:
You are not using this anywhere in the code

Reply:
I have removed it